utils/build.ps1

```diff
@@ -2439,6 +2439,16 @@ function Test-Format {
   }
 }
 
+function Build-LMDB() {
+  Build-CMakeProject `
+    -Src $SourceCache\swift-lmdb `
+    -Bin (Get-HostProjectBinaryCache LMDB) `
+    -Arch $Arch `
+    -Platform Windows `
+    -UseBuiltCompilers C `
```

Comment:
Is there a reason to use `clang-cl` instead of `cl` here?

Reply:
No particular reason. I've updated this patch to use the same approach you took in #71422.

One question about that PR: how close is it to being able to merge? I realize that the next step of this will probably be to get a cmake build of swift-docc working since the goal is to use swift-docc as a dependency to SourceKit-LSP which currently has a cmake build.